### ISSUE TITLE
Some URLs with fragments not working correctly

### ISSUE THREAD

**Issue by TysonMN** [CONNECT-platform/codedoc#98]
In [this post](https://tysonwilliams.coding.blog/2020-07-19_type_parameter_naming) of mine, the URLs to section headings (with URL fragments) are not working.  Similar links are working other posts of mine including headings / fragments containing a space (which is converted to a dash).

![2021-01-30_10-50-44_502](https://user-images.githubusercontent.com/34664007/106362539-0a060200-62e9-11eb-9a41-4a63c78f008e.gif)

**Comment by loreanvictor** [comment]
this is a weird one, since it works fine on safari and firefox, but does not work fine on chrome. other blogposts work on chrome as well, so at least on the first look it is completely unclear what is confusing chrome.

**Comment by TysonMN** [comment]
Did you reproduce this behavior on Chrome?

**Comment by loreanvictor** [comment]
yes. I tried chrome, safari and firefox. chrome is the only one that I can reproduce this bug on. this makes me think there is a possibility that this is actually a chrome bug, which would require further investigation (and honestly so far the chrome team was the worst in terms of responsiveness to bugs we've got on codedoc).
